templates/.github/release-please-config.json

```
@@ -11,23 +11,15 @@
       "type": "fix",
       "section": "Bug Fixes",
       "hidden": false
-    },
-    {
-      "type": "chore",
-      "section": "Miscellaneous",
-      "hidden": false
-    },
-    {
-      "type": "refactor",
-      "section": "Miscellaneous",
-      "hidden": false
     }
```

```suggestion
    }
```

but we do-do want to generate changelog and release for `refactor` since it should end up being a `patch` release

---

What section would you put them under? Refactors?

---

I have been using "Miscellaneous"

Also, is there any way to add cute icons per section?